src/main/java/underdogs/devbie/recommendation/dto/RecommendationResponse.java

```diff
@@ -10,11 +10,13 @@
 
 import lombok.AllArgsConstructor;
 import lombok.Getter;
+import lombok.NoArgsConstructor;
 import underdogs.devbie.recommendation.domain.AnswerRecommendation;
 import underdogs.devbie.recommendation.domain.QuestionRecommendation;
 import underdogs.devbie.recommendation.domain.Recommendation;
 import underdogs.devbie.recommendation.domain.RecommendationType;
 
+@NoArgsConstructor
 @AllArgsConstructor
```

Comment:
```suggestion
@NoArgsConstructor(access = AccessLevel.PROTECTED)
@AllArgsConstructor(access = AccessLevel.PRIVATE)
```
어제 컨벤션 맞춘 것으로 통일하면 좋을 것 같습니다!

Reply:
네 수정했습니다! 감사합니다.